xbmc/interfaces/legacy/Addon.h

```diff
@@ -56,78 +75,145 @@ namespace XBMCAddon
       String getAddonVersion();
 
     public:
-      /**
-       * Addon class.
-       * 
-       * Addon([id]) -- Creates a new Addon class.
-       * 
-       * id          : [opt] string - id of the addon as specified in addon.xml\n
-       * 
-       * *Note, specifying the addon id is not needed.\n
-       *  Important however is that the addon folder has the same name as the addon id provided in addon.xml.\n
-       *  You can optionally specify the addon id from another installed addon to retrieve settings from it.
-       * 
-       * example:
-       *  - self.Addon = xbmcaddon.Addon()
-       *  - self.Addon = xbmcaddon.Addon('script.foo.bar')
-       */
       Addon(const char* id = NULL);
-
       virtual ~Addon();
 
-      /**
-       * getLocalizedString(id) -- Returns an addon's localized 'unicode string'.
-       * 
-       * id        : integer - id# for string you want to localize.
-       * 
-       * example:
-       *   - locstr = self.Addon.getLocalizedString(32000)
-       */
+#ifdef DOXYGEN_SHOULD_USE_THIS
```

Comment:
doxygen can't be adjusted to hide it? `getLocalizedString(...);` is not valid c++, so I'm suspecting this is a hack and that it may break in a future doxygen update.